prime-router/src/main/kotlin/cli/ValidateUtilities.kt

```diff
@@ -0,0 +1,61 @@
+package gov.cdc.prime.router.cli
+
+import com.github.ajalt.clikt.core.CliktError
+import com.github.ajalt.mordant.rendering.TextColors.green
+import com.github.ajalt.mordant.rendering.TextColors.red
+import gov.cdc.prime.router.config.validation.ConfigurationType
+import gov.cdc.prime.router.config.validation.ConfigurationValidationFailure
+import gov.cdc.prime.router.config.validation.ConfigurationValidationResult
+import gov.cdc.prime.router.config.validation.ConfigurationValidationService
+import gov.cdc.prime.router.config.validation.ConfigurationValidationServiceImpl
+import gov.cdc.prime.router.config.validation.ConfigurationValidationSuccess
+import java.io.File
+
+class ValidateUtilities() {
+    companion object ValidateUtilities {
+        val service: ConfigurationValidationService = ConfigurationValidationServiceImpl()
```

Comment:
this should be passed in as a dependency which would mean this logic should not be in a companion object.

```kotlin
class ValidateUtilities(
  val service: ConfigurationValidationService = ConfigurationValidationServiceImpl()
) {
  // ...
}
```

Reply:
I made it consistent with the other utilities from the CLI. Why would it be passed in as a dependency if we want the validation to function the same way from different commands in the CLI?

Reply:
The other example I saw in the CLI project was `CommandUtilities`, which does not take in a service. What Jamie is suggesting is more flexible and more easily unit testable (just pass in a mock when creating a ValidateUtilities object).

My vote is to implement Jamie's suggestion.